Sezai/index.js

```diff
@@ -0,0 +1,39 @@
+// Ex 3
+
+// 0. & 1.
+let snackServings = 33;
+let guests = 25;
+console.log(snackServings, guests);
+
+// 2.
+console.log(snackServings - guests, guests - snackServings);
+
+// 3.
+console.log(snackServings * guests);
+
+// 4.
+console.log(snackServings / guests);
+
+// 5a.
+let drinks = 10;
+resultOne = (snackServings * guests) / drinks;
+console.log(resultOne);
+
+//5b.
+let monkeys = 20;
+let bananas = 5;
+console.log(monkeys / bananas);
+
+// 6.
+let monkeyfights = 20;
+let resultTwo = monkeys + bananas * monkeyfights;
+console.log(resultTwo);
+
+// 7. & 8. see at 5b.
+
```

Comment:
You forgot to answer 8th and 9th tasks! And the solution is :
8th- console.log(++monkeys)
9th- console.log(--bananas)